snuba/datasets/events.py

```diff
@@ -54,6 +54,17 @@ def column_expr(
         parsing_context: ParsingContext,
         table_alias: str = "",
     ):
+        # These column aliases originally existed in the ``discover`` dataset,
+        # but now live here to maintain compatibility between the discover data
+        # model and the events data model that is used for subscriptions. In
+        # the future, these should be resolved at the entity level.
```

Comment:
This will be ok, none is referencing these columns on the events dataset so it cannot break anything. I checked joins as well.